engines/bastion_katello/app/assets/javascripts/bastion_katello/packages/details/package-details-repositories.controller.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @ngdoc object
+ * @name  Bastion.errata.controller:ErrataDetailsRepositoriesController
+ *
+ * @requires $scope
+ * @requires $q
+ * @requires Nutupane
+ * @requires Repository
+ * @requires Environment
+ * @requires ContentView
+ * @requires CurrentOrganization
+ *
+ * @description
+ *   Provides the functionality for the errata details repositories page.
+ */
+angular.module('Bastion.packages').controller('PackageDetailsRepositoriesController',
+    ['$scope', '$q', 'Nutupane', 'Repository', 'Environment', 'ContentView', 'CurrentOrganization',
+    function ($scope, $q, Nutupane, Repository, Environment, ContentView, CurrentOrganization) {
+        var repositoriesNutupane, environment, contentView;
+        var params = {
+            'rpm_id': $scope.$stateParams.packageId,
+            'organization_id': CurrentOrganization
+        };
+
+        repositoriesNutupane = new Nutupane(Repository, params);
+        $scope.detailsTable = repositoriesNutupane.table;
+        $scope.detailsTable.initialLoad = false;
+        repositoriesNutupane.masterOnly = true;
+        repositoriesNutupane.searchKey = 'repositoriesSearch';
```

Comment:
This seems to not be working, I wonder if it broke at some point. The search from the main table is bleeding into the details table.